pkg/metrics/metrics.go

```diff
@@ -59,7 +59,7 @@ var httpCallLatency = promauto.NewHistogramVec( // nolint:gochecknoglobals // Wh
 		Help:    "The latency of the HTTP request, in seconds",
 		Buckets: prometheus.DefBuckets,
 	},
-	[]string{"app", "call", "statuscode", "statuscategory", "kind"}, // Labels
+	[]string{"target", "call", "statuscode", "statuscategory", "kind"}, // Labels
```

Comment:
@jan-ludvik I think we've found why we get two app names separated by comma, if we're right -- this PR will fix the issue. FYI.